element_electrode_localization/coordinate_framework.py

```diff
@@ -52,11 +68,26 @@ class Voxel(dj.Part):
 
 @schema
 class BrainRegionAnnotation(dj.Lookup):
+    """Brain region annotation.
+
+    Attributes:
+        CCF (foreign key): CCF primary key.
+    """
     definition = """
     -> CCF
     """
 
     class BrainRegion(dj.Part):
+        """Brain region information.
+        
+        Attributes:
+            master (foreign key): BrainRegionAnnotion primary key.
```

Comment:
I don't think there's a right way to do this (and I'd also like to automate this down the road, so it'll get rewritten here), but the convention I developed was to write the table name instead of master [e.g.](https://github.com/datajoint/element-deeplabcut/blob/main/element_deeplabcut/model.py#L121) - It looks like you've done the same elsewhere

Reply:
This has been updated to contain the name of the master table